compiler/z/codegen/OMRInstruction.cpp

```diff
@@ -85,7 +85,7 @@ OMR::Z::Instruction::Instruction(TR::CodeGenerator* cg, TR::InstOpCode::Mnemonic
    OMR::Instruction(cg, op, node),
    CTOR_INITIALIZER_LIST
    {
-   TR_ASSERT(cg->getS390ProcessorInfo()->supportsArch(_opcode.getMinimumALS()), "Processor detected does not support instruction %s\n", TR::InstOpCode::getMnemonicName(op));
+   TR_ASSERT(cg->getS390ProcessorInfo()->supportsArch(_opcode.getMinimumALS()), "Processor detected does not support instruction %s\n", cg->getDebug()->getOpCodeName(&this->getOpCode()));
```

Comment:
Note to reviewers: The Travis Lint failure seems to be related  to `(&this->getOpCode..`. 

However the usage of `this` here seems to be consistent with the code around it. So my guess would be that it's not relevant.

Reply:
Will using `self()` work here?

Reply:
Just use `_opcode` to be consistent with the `getMinimalALS` query earlier in the line.